enterprise/frontend/src/metabase-enterprise/rich_text_editing/tiptap/extensions/Command/CommandSuggestion.tsx

```diff
@@ -109,76 +100,18 @@ export const CommandSuggestion = forwardRef<
 >(function CommandSuggestionComponent({ command, editor, query }, ref) {
   const document = useSelector(getCurrentDocument);
   const [selectedIndex, setSelectedIndex] = useState(0);
-  const [showLinkSearch, setShowLinkSearch] = useState(false);
-  const [showEmbedSearch, setShowEmbedSearch] = useState(false);
+
+  const [viewMode, setViewMode] = useState<
+    "linkTo" | "embedQuestion" | "newQuestionType" | null
+  >(null);
+  const [newQuestionType, setNewQuestionType] = useState<
```

Comment:
It would have been the best to move this state to the question type picker menu component, but we cannot control keyboard navigation there as tiptap intercepts some events.